src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -233,100 +335,34 @@ export class Artwork extends React.Component<Props, State> {
         )}
         refreshControl={<RefreshControl refreshing={this.state.refreshing} onRefresh={this.onRefresh} />}
         contentInset={{ bottom: 40 }}
-        keyExtractor={(item, index) => item.type + String(index)}
```

Comment:
now that the sections have a `key` property, `FlatList` will find it there without our intervention.